src/components/base/button/index.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import propTypes from "prop-types";
+import styles from "components/base/button/index.module.scss";
+import { TYPE_SUBMIT } from "utils/const";
+
+function Button({ children, onClick, className, buttonType }) {
+  return (
+    <button
+      className={`${styles.button} ${className}`}
+      onClick={onClick}
+      type={buttonType}
+    >
+      {children}
+    </button>
+  );
+}
+
+Button.propTypes = {
+  children: propTypes.oneOfType([propTypes.string, propTypes.node]).isRequired,
```

Comment:
node covers string too. you can just write `propTypes.node` . That will work as well